Library/Formula/cucumber-cpp.rb

```diff
@@ -0,0 +1,24 @@
+class CucumberCpp < Formula
+  desc "Support for writing Cucumber step definitions in C++"
+  homepage "https://cucumber.io"
+  url "https://github.com/cucumber/cucumber-cpp/archive/1e633c6b59c75b959b480dcc4e40123e7fc59032.tar.gz"
+  version "0.3"
+  sha256 "1e211db8b8c230f291c8774d1c2e2bbbcf264d92016fd6081557425e43569c19"
+
+  depends_on "cmake" => :build
+  depends_on "ruby"
+
+  def install
+    system "gem", "install", "bundler"
+    system "cmake", "-DCUKE_DISABLE_GTEST=on",
+                    "-DCUKE_DISABLE_CPPSPEC=on",
+                    "."
+    system "cmake", "--build", "."
+    include.install "include/cucumber-cpp"
+    lib.install Dir["src/*.a"]
+  end
+
+  test do
+    system "false"
```

Comment:
```
brew install --interactive --debug cucumber-cpp
/usr/bin/irb
puts Gem.path
```

So one way is to put the resource in the make-shift home directory, which should work.

For <= Mountain Lion you'd want to skip the test, or make homebrew ruby a hard dependency for <= Mountain Lion, since cucumber requires ruby 1.9.3